docs/pages/User/GettingStarted/demo_k8s_on_desktop.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Demo/Tutorial with KubernetesOnDesktop
+weight: 4
+---
+
+## Introduction
+In this demo/tutorial we will show you how to install [liqo.io](https://liqo.io) on two [k3s](https://k3s.io/) clusters from scratch and then run a real application ([KubernetesOnDesktop](#about-kubernetesondesktop)) by using the foreign cluster.
+
+The goal is to offload the (blender) application execution from a cluster to another one (e.g. to cluster that has a NVIDIA graphic card). See the schema below: 
+
+![](/images/k8s-on-desktop-demo/introduction_schema.svg)
+
+To be more specific, we will try to execute a blender `pod` in a *foreign cluster* (that is represented in the *local cluster* as a *virtual node* named `liqo-<...>`) and a viewer `pod` in the *local cluster*.
+
+N.B.:
+From now on, when we'll talk about "*local cluster*" we'll refer to the one that will run the `cloudify` script ([see afterwards](#about-kubernetesondesktop)), and when we'll talk about "*foreign cluster*" we'll refer to the other one.
+
+## Installation of the required software
+To install all the required software we need to follow this steps:
+
+1. [Install k3s](#install-k3s) in both clusters;
+2. [Install liqo.io](#install-liqo-io) in both clusters; 
+3. [Install KubernetesOnDesktop](#install-kubernetesondesktop) just in one of the clusters (the so called *local cluster*).
+
+### Install k3s
+Assuming you already have two linux (we tested it with Ubuntu Desktop 20.04 LTS) machine or Virtual Machine up and running in the same subnet, we can install [k3s](https://k3s.io/) by using the official script as you can see in [K3s Quick-Start Guide](https://rancher.com/docs/k3s/latest/en/quick-start/). So, you only need to run the following:
+
+```bash
+curl -sfL https://get.k3s.io | sh -
+```
+
+When the script ends, to make [liqo.io](https://liqo.io) work properly, you need to modify the `/etc/systemd/system/k3s.service` file by adding the `--kube-apiserver-arg anonymous-auth=true` service execution parameter. You can do this by executing the following:
+
+```bash
+sudo sed -i "s#server#server --kube-apiserver-arg anonymous-auth=true#" /etc/systemd/system/k3s.service
+```
+
+After this operation, your `k3s.service` file should look like this:
```

Comment:
An [expand](https://learn.netlify.app/en/shortcodes/expand/) could be useful here.

Reply:
Fixed